app/assets/stylesheets/govuk-component/_lead-paragraph.scss

```diff
@@ -0,0 +1,6 @@
+.app-c-lead-paragraph {
```

Comment:
The prefix needs to be 'pub' so it doesnt conflict with the app level component

Reply:
Fixed